Resources/Public/JavaScript/find.js

```diff
@@ -267,19 +267,19 @@ var tx_find = (function () {
      *
      * @param {object} ranges
      */
-    var selectRanges = function (ranges) {
+    var selectRanges = function (event, ranges) {
       var newRange = roundedRange(ranges.xaxis);
       plot.setSelection({'xaxis': newRange}, true);
       hideTooltip();
-      startSearchWithNewFacet(newRange);
+      startSearchWithNewFacet(event, newRange);
     };
 
     jGraphDiv.bind('plotclick', function (event, pos, item) {
       return true;
     });
 
-    jGraphDiv.bind('plotselected', function (event, ranges) {
-      selectRanges(ranges);
+    jGraphDiv.one('plotselected', function (event, ranges) {
```

Comment:
`one` or `on`?

Reply:
intentionally »one« as an incorrect event seems to be fired after the correct one and leads to an incorrect parameter (see the commit message)